FactoryTest/libraries/ButtonDebounce/examples/simple_callback/simple_callback.ino

```diff
@@ -0,0 +1,15 @@
+#include <ButtonDebounce.h>
```

Comment:
Did we copy in a library, if so did we pull the license along with it?

Reply:
Yes, I copied the entire library folder for ButtonDebounce and FanController, including license, readme, examples, etc.